SharpBgfx/Bgfx.cs

```diff
@@ -994,7 +982,7 @@ class DefaultCallbackHandler : ICallbackHandler {
                 if (errorType == ErrorType.DebugCheck)
                     Debug.Write(message);
                 else {
-                    Debug.Write(string.Format("{0}: {1}", errorType, message));
+                    Debug.Write($"{fileName} ({line})  {errorType}: {message}");
```

Comment:
Some people who use this library compile for older .NET or C# versions, so using interpolated strings would break for them.